K/kdb_c_api/build_tarballs.jl

```diff
@@ -44,7 +44,7 @@ platforms = [
 
 # The products that we will ensure are always built
 products = [
-    LibraryProduct("c", :kdb_c_so)
+    LibraryProduct("c", :kdb_c_dl)
```

Comment:
Also, note that changing the name of the variable referencing the shared library is a breaking change, I'm not sure why you'd want to do that

Reply:
reverted to previous. right now my WIP package Q2.jl is the only one to use it, so I don't think there was anything to break to begin with